BSPConvert.Lib/Source/EntityConverter.cs

```diff
@@ -228,6 +234,22 @@ private static void OpenDoorOnPressed(Entity button, Entity door)
 			button.connections.Add(connection);
 		}
 
+		private void TargetSpeedOnPressed(Entity button, Entity targetSpeed)
```

Comment:
This method has a lot of shared code with `ConvertTargetSpeedTrigger()`. You can refactor this into a single method called `FireTargetSpeedOnOutput(Entity activator, Entity targetSpeed, string output)` where `output` sets the name on `EntityConnection`.